src/Runner.Common/JobServerQueue.cs

```diff
@@ -30,6 +31,7 @@ public sealed class JobServerQueue : RunnerService, IJobServerQueue
         private static readonly TimeSpan _delayForWebConsoleLineDequeue = TimeSpan.FromMilliseconds(500);
         private static readonly TimeSpan _delayForTimelineUpdateDequeue = TimeSpan.FromMilliseconds(500);
         private static readonly TimeSpan _delayForFileUploadDequeue = TimeSpan.FromMilliseconds(1000);
+        private static readonly TimeSpan _delayForResultsSummaryUploadDequeue = TimeSpan.FromMilliseconds(1000);
```

Comment:
I would rename everything in this file to not include `Result`, `IJobServerQueue` doesn't need to know it's uploading stuff to ResultService.